### DISCUSSION TITLE
how can i do a zombifier? or an iron farm?

### DISCUSSION THREAD

**Discussion by RealDoctorTerror** [Luke100000/minecraft-comes-alive#843] [General]
I tried to make a zombifier, it doesnt work because the zombie kills the villager, and the villager defend against the zombie. the villager kills my zombie in the iron farm. Can I disable villager fire mobs?

**Comment by Luke100000** [comment]
The point of this mod is to make the villagers more realistic. If a zombie walks toward you and you can not flee, you punch it. No, you can not disable this :)
